Modules/_datetimemodule.c

```diff
@@ -5514,19 +5517,27 @@ datetime_utcfromtimestamp(PyObject *cls, PyObject *args)
 static PyObject *
 datetime_strptime(PyObject *cls, PyObject *args)
 {
-    static PyObject *module = NULL;
```

Comment:
Ah, this one definitely slipped through the cracks.  Thanks for taking care of it.